src/ActionList/Item.tsx

```diff
@@ -55,6 +55,8 @@ export type ActionListItemProps = {
    * Is the `Item` is currently selected?
    */
   selected?: boolean
+  // TODO: Document `active` prop
+  active?: boolean
```

Comment:
Note: I coupled the `active` styles with an `active` prop instead of the `aria-current` attribute. I think there are cases where we would want to an item to have active styles but it isn't necessarily appropriate to use `aria-current`. @siddharthkp please correct me if I'm wrong.

Reply:
Fair enough! Can you talk more about the cases where this would happen?